packages/harpguru-core/src/components/option-title-stack/option-title-stack.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+
+import { OptionTitle } from '../option-title'
+import type { OptionStackProps } from '../option-stack'
+import type { WithTransition } from '../../types'
+import { useInterpolateOptionStackTransitionValue } from '../../hooks'
```

Comment:
This is the second reference to this I've seen in this review so that means the previous point about wanting to localise it can be discarded.